site/content/en/docs/getting_started/overview.md

```diff
@@ -0,0 +1,209 @@
+| [**Frequently Asked Questions**](/cvat/docs/faq/)                                              | This section addresses common queries and provides helpful answers and insights about using CVAT.                                                                                                                          |
+
+<!--lint enable maximum-line-length-->
+
+## Integrations
+
+CVAT is a global tool, trusted and utilized by teams worldwide.
+Below is a list of key companies that contribute significantly to our
+product support or are an integral part of our ecosystem.
+
+> **Note:** If you're using CVAT, we'd love to
+> hear from you at [contact@cvat.ai](mailto:contact+github@cvat.ai).
+
+<!--lint disable maximum-line-length-->
+
+| Integrated Service                                                                                     | Available In          | Description                                                                                                                                                                                                                                     |
+| ------------------------------------------------------------------------------------------------------ | --------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [**Human Protocol**](https://hmt.ai)                                                                   | Cloud and Self-hosted | Incorporates CVAT to augment annotation services within the Human Protocol framework, enhancing its capabilities in data labeling.                                                                                                              |
+| [**FiftyOne**](https://fiftyone.ai)                                                                    | Cloud and Self-hosted | An open-source tool for dataset management and model analysis in computer vision, FiftyOne is [closely integrated](https://voxel51.com/docs/fiftyone/integrations/cvat.html) with CVAT to enhance annotation capabilities and label refinement. |
+| [**Toloka**](https://toloka.ai)                                                                        | Cloud                 | Utilizes CVAT for crowdsourced data labeling and annotation services, enriching Toloka's diverse task handling capabilities. For more information, see [**Integration with Toloka**](/docs/integration/toloka/).                                |
+| [**Hugging Face and Roboflow**](https://www.cvat.ai/post/integrating-hugging-face-and-roboflow-models) | Cloud                 | In CVAT Cloud, models from [**Hugging Face**](https://huggingface.co/) and [**Roboflow**](https://roboflow.com/) can be added to enhance computer vision tasks.                                                                                 |
```

Comment:
I'm not sure why all previous links were to services, and this one to documentation?

Reply:
@nmanovic it is the only place where integration is described. There are links to the services in the description.

Reply:
UPD: nevermind my previous comment, I ll redo it the same way as Toloka